--- /home/kjetil/src/4store-v1.1.5/src/frontend/query.c	2012-06-11 11:47:24.000000000 +0200
+++ /home/kjetil/src/4store-bgp-slow/4store-v1.1.5/src/frontend/query.c	2013-05-01 16:07:52.458402204 +0200
@@ -25,6 +25,8 @@
 #include <glib.h>
 #include <pthread.h>
 #include <rasqal.h>
+#include <unistd.h>
+
 
 #include "4store-config.h"
 #include "query.h"
@@ -915,6 +917,7 @@
          * correct relative order, otherwise OPTIONAL blocks which share variables
          * give the wrong result */
         for (int j=start; j<=q->block; j++) {
+	    sleep(2); /* Inserted by Kjetil Kjernsmo for a experiment */
             if (!q->bb[j]) {
 #ifdef DEBUG_MERGE
                 printf("skipping B%d, no bindings\n", j);
