--- /home/kjetil/src/4store-v1.1.5/src/frontend/filter.c	2012-06-11 11:47:24.000000000 +0200
+++ /home/kjetil/src/4store-filter-slow/4store-v1.1.5/src/frontend/filter.c	2013-05-02 00:15:41.546174768 +0200
@@ -24,6 +24,7 @@
 #include <time.h>
 #include <errno.h>
 #include <uuid/uuid.h>
+#include <unistd.h>
 
 #include "filter-datatypes.h"
 #include "filter.h"
@@ -983,6 +984,8 @@
 
 fs_value fn_lang_matches(fs_query *q, fs_value l, fs_value p)
 {
+    usleep(2000); /* Inserted by Kjetil Kjernsmo for a experiment */
+
     if (fs_is_error(l)) {
 	return l;
     }
